test/Microsoft.AspNet.Razor.Test/TagHelpers/TagHelperParseTreeRewriterTest.cs

```diff
@@ -19,6 +19,832 @@ namespace Microsoft.AspNet.Razor.Test.TagHelpers
 {
     public class TagHelperParseTreeRewriterTest : CsHtmlMarkupParserTestBase
     {
+        public static TheoryData OptOutPartialTagDataCSharp
```

Comment:
- are warnings for `"@{<!p class=\"btn\" >}"` (missing end tag) tested elsewhere?
- either way, should probably test `"@{<!p class=\"btn\" /}" as well